package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.34",
+  "publisher": "Aqovia",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
+    "hnw.vscode-auto-open-markdown-preview",
+    "redhat.vscode-yaml",
+    "Zignd.html-css-class-completion",
+    "streetsidesoftware.code-spell-checker",
+    "ms-dotnettools.csharp",
+    "donjayamanne.git-extension-pack",
+    "eriklynd.json-tools",
+    "dbaeumer.vscode-eslint",
+    "mrmlnc.vscode-less",
+    "DavidAnson.vscode-markdownlint",
+    "ms-mssql.mssql",
+    "leizongmin.node-module-intellisense",
+    "christian-kohler.npm-intellisense",
```

Comment:
`leizongmin.node-module-intellisense` appears to duplicate functionality of `christian-kohler.npm-intellisense`. I don't think we should install both. Maybe ask in Slack (#eng-web-components might be the most appropriate channel) for feedback on which one is better?

Reply:
Christian Kohler's version is more widely used and maintained. Removed the other one.